pkg/tbtc/signing.go

```diff
@@ -64,11 +65,16 @@ type signingAttemptParams struct {
 // signingAttemptFn represents a function performing a signing attempt.
 type signingAttemptFn func(*signingAttemptParams) (*signing.Result, error)
 
+// waitForSigningAttemptFn represents a function blocking the attempt execution
+// until the given block height.
+type waitForSigningAttemptFn func(context.Context, uint64) error
```

Comment:
Same naming concerns as in my previous comment for `waitForDkgAttemptFn`.

Reply:
d34a5741576431e7f45ad6a8b50a25f88f380cd3